.gitignore

```diff
@@ -1,31 +1,23 @@
+# IDE directories
 .idea
 .vscode
+
+# macOS hidden files
 .DS_Store
 
-/beta
-/build
+# XCode project files
 Transmission.xcodeproj/*.xcworkspace
 Transmission.xcodeproj/xcuserdata
-/cli/transmissioncli
-/daemon/transmission-daemon
-/daemon/transmission-proxy
-/daemon/transmission-remote
-/gtk/defines.h
-/gtk/transmission-gtk
-/libtransmission/*.a
-/libtransmission/version.h
+
+# files Xcode creates
 /macosx/Info.plist
-/macosx/**/*~.nib
-node_modules/
-/po/*.mo
-/REVISION
-/tests/**/*.out
-/third-party/miniupnp/miniupnpcstrings.h
-/third-party/suffixes_dafsa.h
-/web/public_html/transmission-app.js.map
+REVISION
```

Comment:
this one is created by cmake, not Xcode

Reply:
I thought so too, but @nevack said xcode made it on his system?